tools/clang/test/CodeGenSPIRV/inline-spirv/spv.raytracing.hlsl

```diff
@@ -0,0 +1,22 @@
+// RUN: %dxc %s -T cs_6_8 -spirv -fspv-target-env=vulkan1.3 -E main -O0 | FileCheck %s
+
+// CHECK-DAG: OpCapability RuntimeDescriptorArray
+// CHECK-DAG: OpCapability RayQueryKHR
+
+// CHECK-DAG: OpDecorate %MyScene DescriptorSet 1
+// CHECK-DAG: OpDecorate %MyScene Binding 3
+
+using A = vk::SpirvOpaqueType</* OpTypeAccelerationStructureKHR */ 5341>;
+// CHECK: %[[name:[^ ]+]] = OpTypeAccelerationStructureKHR
+
+using RA [[vk::ext_capability(/* RuntimeDescriptorArray */ 5302)]] = vk::SpirvOpaqueType</* OpTypeRuntimeArray */ 29, A>;
+// CHECK: %[[rarr:[^ ]+]] = OpTypeRuntimeArray %[[name]]
+
+// CHECK: %[[ptr:[^ ]+]] = OpTypePointer UniformConstant %[[rarr]]
+// CHECK: %MyScene = OpVariable %[[ptr]] UniformConstant
+[[vk::binding(3, 1)]]
+[[vk::ext_storage_class(0)]]
+RA MyScene;
```

Comment:
I question whether this should work. We don't have a formal spec for vk::binding, but in the idea of whether or not it is valid and meaningful is tied to the type of the variable. In this case, RA is not a resource type. I could image this not working in Clang where we defined what it could apply to more strictly, I believe.